webrender_traits/src/channel_ipc.rs

```diff
@@ -19,9 +21,16 @@ pub type PayloadSender = IpcBytesSender;
 
 pub type PayloadReceiver = IpcBytesReceiver;
 
-impl PayloadHelperMethods for PayloadSender {
-    fn send_vec(&self, data: Vec<u8>) -> Result<(), Error> {
-        self.send(&data)
+impl PayloadSenderHelperMethods for PayloadSender {
+    fn send(&self, data: Payload) -> Result<(), Error> {
+        self.send(&data.to_data())
```

Comment:
Yes. send(Vec) is still message based. So the receiver expects to get everything in recv call. In the long term we'd probably want to fix this to be more efficient, but it looks like ipc-channel has lower hanging efficiency problems to deal with first.